File: projects/10/ruby/FlyingBird.rb
# flying_bird.rb defines the behavior of a bird that flies.
# Author: David Yoo (gy24)
# Date: Mar 29, 2022

require './Bird.rb'

class FlyingBird < Bird
  # Get the movement of this bird.
  def movement
    " flew past"
  end
end

File: projects/10/java/FlyingBird.java
/**
 * FlyingBird.java defines the flying bird
 *
 * Created by: David Yoo (gy24)
 * Date: Mar 29, 2022
 */

public class FlyingBird extends Bird {
    /**
     * implicit constructor
     */
    public FlyingBird() {
	    super();
    }

    /**
     * explicit constructor
     */
    public FlyingBird(String name) {
	    super(name);
    }

    /**
     * Get this Bird's movement.
     */
    public String movement() {
	    return "flew past";
    }
}